helm/prometheus-agent/charts/prometheus-agent/templates/servicemonitor.yaml

```diff
@@ -15,24 +15,24 @@ spec:
     matchNames:
       - {{ .Release.Namespace }}
   endpoints:
-  - interval: 30s
+  - interval: {{ .Values.serviceMonitor.scrapeInterval }}
```

Comment:
Isn't it `.Values.scrapeInterval` instead ?

Reply:
Actually yes https://github.com/giantswarm/prometheus-agent-app/pull/55